opentelemetry-sdk/src/metrics/instrument.rs

```diff
@@ -169,12 +169,15 @@ pub struct Stream {
     pub unit: Unit,
     /// Aggregation the stream uses for an instrument.
     pub aggregation: Option<Aggregation>,
-    /// applied to all attributes recorded for an instrument.
-    pub attribute_filter: Option<Filter>,
+    /// An allow-list of attribute keys that will be preserved for the stream.
+    ///
+    /// Any attribute recorded for the stream with a key not in this set will be
+    /// dropped.
+    ///
+    /// If this set is empty or `None`, all attributes will be kept.
```

Comment:
what would be the way to drop every attribute?

Reply:
hm good point, maybe empty set should be drop all